### PR TITLE
intro completed

### PR DESCRIPTION
**Pull request by DilsherB**
intro completed and merged in main

### CODE REVIEW

File: READme.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Hi, I'm Dilsher Balouch.
+I am learning "software development" at Microverse in the batch starting from 28th Nov. 2022.
+This is my portfolio to show case my abilities in the field of software development.
```
Comment (andgarzonmal) [inline READme.md]: - [ ]  Kindly add a professional readme file to your project, you can use the one from microverse [README template](https://github.com/microverseinc/readme-template)

File: index.html [hunk 2]
```diff
@@ -4,9 +4,35 @@
...
+            <a href="#"><img src="images/github.svg" alt=""></a>
+            <a href="#"><img src="images/hand.svg" alt=""></a>
+        </div>
+    </section>
+    <section id="buttons">
+        <button id="enabled">Enabled</button>
+        <button id="hover">Hover</button>
+        <button id="pressed">Pressed</button>
+        <button id="disabled">Disable</button>
+    </section>
```
Comment (andgarzonmal) [inline index.html]: Kindly stick to the Figma template as much as you can, please notice in the Figma template there are no buttons enabled, hover, pressed, or disabled.

## Figma design 
image

## Your design 

image

File: index.html [hunk 3]
```diff
@@ -4,9 +4,35 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
     <title>Dilsher Balouch</title>
 </head>
 <body>
-    
+    <nav id="header">
+        <div id="logo">Dilsher Balouch</div>
```
Comment (andgarzonmal) [inline index.html]: Kindly use an `a` tag on your logo, This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

File: index.html [hunk 4]
```diff
@@ -4,9 +4,35 @@
...
+            don’t hestiate to contact me.
+        </p>
+        <div id="sociHeading">LET'S CONNECT</div>
+        <div id="sociConnector">
+            <a href="#"><img src="images/twitter.svg"></a>
+            <a href="#"><img src="images/linkedin.svg"></a>
+            <a href="#"><img src="images/fb.svg" alt=""></a>
+            <a href="#"><img src="images/github.svg" alt=""></a>
+            <a href="#"><img src="images/hand.svg" alt=""></a>
+        </div>
```
Comment (andgarzonmal) [inline index.html]: - [x]  Kindly use a `ul` tag to wrap all of your social media icons, also remember as these icons will be linked to your social media use an `a` tag on each icon.

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi @DilsherB    ✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 3 above)
Comment (DilsherB) [reply index.html]: @andgarzonmal 
Thank you for your review and detailed explanation. I have done all these changes.

File: index.html (hunk 4 above)
Comment (DilsherB) [reply index.html]: @andgarzonmal 
Thnaks a lot for detailed review/description. I have done all the changes mentioned by you. Please guide me whether I have to submit it again for review or not.

File: index.html [hunk 5]
```diff
@@ -4,9 +4,29 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
     <title>Dilsher Balouch</title>
 </head>
 <body>
-    
+    <nav id="header">
+        <a href="#logo" id="logo">Dilsher Balouch</a>
+        <div id="menuIcon"><img src="images/menuIcon.svg" alt="Mobile Menu Icon"></div>
+    </nav>
```
Comment (brainconnect93) [inline index.html]: - [x] Hi @DilsherB, well done to have worked on your project this far, I would kindly suggest you wrap your `<nav>` tag inside an `<header>` tag. Typically the `<header>` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

Hint on what am driving @: 👇 
```HTML
<header>
    <a href="#logo" id="logo">Dilsher Balouch</a>
    <nav>
       
    </nav>
</header>
```

File: index.html [hunk 6]
```diff
@@ -4,9 +4,29 @@
...
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+        <div id="sociHeading">LET'S CONNECT</div>
+        <ul id="sociConnector">
+            <li><a href="https://twitter.com/_brilliantMindz"><img src="images/twitter.svg"></a></li>
+            <li></li><a href="https://www.linkedin.com/in/dilsher-balouch-7139432a/"><img src="images/linkedin.svg"></a></li>
+            <li></li><a href="https://www.facebook.com/dilsherb"><img src="images/fb.svg" alt=""></a></li>
+            <li></li><a href="https://github.com/DilsherB"><img src="images/github.svg" alt=""></a></li>
+            <li></li><a href="#"><img src="images/hand.svg" alt=""></a></li>
```
Comment (brainconnect93) [inline index.html]: - [x] I would suggest you check the closing tag of your `</li>` it's conflicting with each other. Kindly check it and rectify

File: style.css [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+  background-repeat: no-repeat;
+}
+
+#introHeader {
+  color: #172b4d;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+}
```
Comment (brainconnect93) [inline style.css]: - [ ] I would suggest we stick to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=t4fpDJvaCtvytWN0-0) as suggested by the previous reviewer. 

- This is the Figma template 👇 

real

- And this is what you have in your project 👇 

temp1

- Kindly follow the highlighted arrows to see what changes I am requesting for

1.  Kindly make the content of the project align with the title as seen in the template, also there should be some `padding left` in order for them to have some spaces away from the wall
2. Kindly check your social media icons too and fix their alignment, also remove the dot sign showing around the icons. You can simply do that by giving the list some styling value of `<list-style-type: none;>`

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @DilsherB  , 👋 

# STATUS: Changes Required :recycle:

Good job so far in implemented some of the changes the previous reviewer request for!
There are some issues that you still need to work on to go to the next project but you are almost there!




### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 7 above)
Comment (DilsherB) [reply style.css]: @brainconnect93 
Thanks for your feedback.
I have done all but unable to align the social media icons. **_Can you help me in this?_**

**Comment by DilsherB** [comment]
Social media alignment is done

**Review by mavericksbalitaan — APPROVED** [review]
Hi **@DilsherB** 😃 , 

# Approved 👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Required changes implemented ✅
- Have use the right GitHub flow ✅
- Professional README.md file ✅
- Linter checks passed ✅

## Optional suggestions

- **[OPTIONAL]**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Code review by **Mavericks De Leon Balitaan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**